gemrb/core/DisplayMessage.h

```diff
@@ -36,13 +36,7 @@
 
 namespace GemRB {
 
-static constexpr Color DMC_WHITE {0xf0, 0xf0, 0xf0, 0xff};
-static constexpr Color DMC_RED = ColorRed;
-static constexpr Color DMC_LIGHTGREY {0xd7, 0xd7, 0xbe, 0xff};
-static constexpr Color DMC_BG2XPGREEN {0xbc, 0xef, 0xbc, 0xff};
-static constexpr Color DMC_GOLD {0xc0, 0xc0, 0x00, 0xff};
-static constexpr Color DMC_DIALOG = DMC_WHITE;
-static constexpr Color DMC_DIALOGPARTY = {0x80, 0x80, 0xff, 0xff};
+enum { DMC_DIALOG, DMC_DIALOGPARTY, DMC_GOLD, DMC_RED, DMC_BG2XPGREEN, DMC_LIGHTGREY, DMC_WHITE };
```

Comment:
Ok, so now that things have settled, it's time for a big rename (but no need to jump at this immediately). 
First make this a named enum class (see ieInventoryType for an example), then remove the DMC_ prefix from the value names.
BG2XPGREEN is a shitty name as well, I propose you replace it with XPCHANGE.
Then you can also change the functions taking these constants as integer parameters to now take them as parameters of this new type (whatever you named the enum).
These things will force you to change all the existing users of the DMC constants (that's why we waited), which is easy to resolve with search&replace.